index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Richard's Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link href="https://fonts.googleapis.com/css2?family=Anton&family=Inter&family=Permanent+Marker&display=swap" rel="stylesheet">
+  </head>
+  <body>
+
+   <div class="app-wrap">
+     <header class="app-header">
+         <a >Welcome</a>
+
+         <nav><i class="fas fa-bars"></i></nav>
+     </header>
+     <!-- <div class="for_background"> -->
+
+       <section class="headline">
+         <div class="primary">
+            <h1>Hey there. I'm Richard</h1>
+
+            <p>I'm a software developer</p>
+         </div>
+          <div class="support">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+  don’t hestiate to contact me.</p>
+          </div>
+          <div class="social-media">
+            <a href="#"><i class="fab fa-twitter social"></i></a>
+            <a href="#"><i class="fab fa-linkedin social"></i></a>
+            <a href="#"><i class="fab fa-medium social"></i></a>
+            <a href="#"><i class="fab fa-github social"></i></a>
+            <a href="#"><i class="fab fa-angellist social"></i></a>
+
+          </div>
+
+          <div class="arrow">
+            <button  class="arrow-button" ><i class="fas fa-chevron-circle-down fa-3x"></i></button>
+          </div>
+        </section>
+
+     <!-- </div> -->
+
+
+   </div>
+
+
+  </body>
```

Comment:
- [ ] Hello, it is essential you follow to stick to the Figma you chose. The design doesn't have any blue background colour nor it does have a margin in the body element. I suggest you remove the blue background-colour and follow your chosen design:

Figma Design | Your Design  |
:-----------------:|:-----------------:|
![d1](https://user-images.githubusercontent.com/39523312/148354998-e4d37fa1-e21f-45b5-8903-e720d1730436.png) |![d2](https://user-images.githubusercontent.com/39523312/148355010-f19bd39c-d1d3-4ebe-9bb4-31cf45b45c5d.png)

Reply:
If I may ask, how exactly is the mobile section distinguished from the rest of the page? I had the background to show that distinction. See the attache
<img width="1440" alt="Screenshot 2022-01-06 at 12 45 49" src="https://user-images.githubusercontent.com/35674623/148363397-5af18769-99fc-47f7-b23c-daae7fa68a2a.png">
d